__generator__/builtin.yml

```diff
@@ -1221,14 +1221,14 @@ ratelimit.check_rate:
   reference: "https://developer.fastly.com/reference/vcl/functions/rate-limiting/ratelimit-check-rate/"
   on: [RECV, HASH, HIT, MISS, PASS, FETCH, ERROR, DELIVER, LOG]
   arguments:
-    - [STRING, ID, INTEGER, INTEGER, INTEGER, ID, TIME]
+    - [STRING, ID, INTEGER, INTEGER, INTEGER, ID, RTIME]
```

Comment:
Fastly document says the sixth argument type of ttl must be `TIME` type.
https://developer.fastly.com/reference/vcl/functions/rate-limiting/ratelimit-check-rate/
What does this change intend to do?

Reply:
I reached out to Fastly and they confirmed it is a documentation bug. The value should be RTIME